superset-frontend/package.json

```diff
@@ -321,27 +322,26 @@
     "jest-html-reporter": "^3.10.2",
     "jest-websocket-mock": "^2.5.0",
     "jsdom": "^24.0.0",
-    "lerna": "^8.1.5",
+    "lerna": "^8.1.7",
     "less": "^4.2.0",
-    "less-loader": "^10.2.0",
+    "less-loader": "^12.2.0",
     "mini-css-extract-plugin": "^2.9.0",
     "mock-socket": "^9.3.1",
     "node-fetch": "^2.6.7",
     "po2json": "^0.4.5",
-    "prettier": "3.1.0",
+    "prettier": "3.3.3",
     "prettier-plugin-packagejson": "^2.4.10",
     "process": "^0.11.10",
     "react-resizable": "^3.0.5",
     "react-test-renderer": "^16.14.0",
     "redux-mock-store": "^1.5.4",
-    "sinon": "^9.0.2",
+    "sinon": "^18.0.0",
```

Comment:
That's quite a bump!!

Reply:
Plenty of accompanying refactors though 😄